docs-raw/site/docs/fabric-gateway-features.md

```diff
@@ -273,6 +274,24 @@ spec:
           - "spp-application.write"
 ```
 
+### External Service Management
+It is possible to integrate the Gateway Operator with other operators which manage the creation of [Kubernetes services](https://kubernetes.io/docs/concepts/services-networking/service/). The `x-external-service-provider` key should be set as per the below example.
+```yaml
+apiVersion: zalando.org/v1
+kind: FabricGateway
+metadata:
+  name: stackset-managed-test
+spec:
+  x-external-service-provider:
+    hosts:
+      - stackset-managed-test.cluster.zalan.do
+    stackSetName: <name-of-stackset-k8s-resource>
+  paths:
+    /resources:
+      get: {}
+```
+In the above example, we are not defining any services (and it is illegal to define services via the `x-fabric-service` key if you have the `x-external-service-provider` set) in the Gateway resource. Instead, we are saying that these services will be provided by a [StackSet](https://cloud.docs.zalando.net/tutorials/migrating-from-stups/#recommended-stacksets) operator. The gateway will initially be created in a state where no ingress resources are generated. It will remain like this until it finds the named StackSet and reads the service naming details from it.
```

Comment:
The created gateway will not generate any ingress resources until the named StackSet is present and has service details available.